test/semantics/doconcurrent01.f90

```diff
@@ -49,11 +65,22 @@ subroutine do_concurrent_test2(i,j,n,flag)
   use iso_fortran_env, only: team_type
   implicit none
   integer :: i, n, flag, flag2
+  logical :: halting
   type(team_type) :: j
   do concurrent (i = 1:n)
     change team (j)
+!ERROR: call to impure subroutine in DO CONCURRENT not allowed
+!ERROR: IEEE_GET_FLAG not allowed in DO CONCURRENT
```

Comment:
These symbols aren't coming from the intrinsic module, so they shouldn't  trigger errors that are particular to intrinsic modules.

Reply:
Will there be a way to distinguish between the actual intrinsic modules and use defined modules with the same names?

Reply:
Yes, we have to support `USE, INTRINSIC` and `USE, NON_INTRINSIC` too.